### PR TITLE
disableError()

### PR DESCRIPTION
**Pull request by enVolt**
An optional function, which will completely disable any error. (`undefined` is returned, instead of any error).

* Test Case
* version changed to 0.7.0
* updated `README.md`
* updated `package.json`

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -122,8 +122,35 @@ if (process.env.NODE_ENV === 'production') {
 }
 ```
 
+### env.disableError()
```
Comment (ctavan) [inline README.md]: Could you call this `disableErrors()` so that the wording is consistent with `disableFallbacks()`?

File: package.json [hunk 2]
```diff
@@ -6,9 +6,10 @@
     "Moritz von Hase",
     "Jonas Dohse <jonas@dohse.ch>",
     "Jan Lehnardt",
-    "Tim Ruffles <timruffles@gmail.com>"
+    "Tim Ruffles <timruffles@gmail.com>",
+    "Ashwani Agarwal <ashwani.a@outlook.com>"
   ],
-  "version": "0.6.0",
+  "version": "0.7.0",
```
Comment (ctavan) [inline package.json]: Please don't bump the npm version yet. I will do that in a separate commit before publishing to npm.

**Comment by ctavan** [comment]
@enVolt thanks for this great contribution! I had two minor comments. Would you mind fixing them (and squashing them into this commit)? I'd then be happy to merge and publish.

**Comment by enVolt** [comment]
@ctavan Thanks, this is my first ever contribution, so I was kind of nervous.

I kinda lost you `squashing them into this commit`, do you want me to recreate pull request, with single commit change?

**Comment by ctavan** [comment]
Just keep this pull-request but *amend* your existing commit, then force push... In practice:

```
# do your changes
git add *
git commit --amend
git push -f
```

**Comment by enVolt** [comment]
Changes

* reverted Change Log
* package.json version

**Comment by enVolt** [comment]
@ctavan ?

File: README.md [hunk 3]
```diff
@@ -122,6 +122,30 @@ if (process.env.NODE_ENV === 'production') {
 }
 ```
 
+### env.disableErrors()
+
+`getenv` won't throw any error. If a fallback value is provided, that will be returned, else `undefined` is returned.
+
+```javascript
+getenv.disableErrors();
+console.log(getenv("RANDOM");
```
Comment (ctavan) [inline README.md]: There's a closing `)` missing.

File: README.md [hunk 4]
```diff
@@ -122,6 +122,30 @@ if (process.env.NODE_ENV === 'production') {
...
+// undefined
+```
+
+### env.enableErrors()
+
+Revert the effect of `disableErrors()`.
+
+```javascript
+getenv.disableErrors();
+console.log(getenv("RANDOM");
```
Comment (ctavan) [inline README.md]: See above.

File: test/fallbacks.js [hunk 5]
```diff
@@ -13,6 +13,22 @@ tests['getenv.disableFallbacks() makes relying on fallbacks an error'] = functio
   getenv.enableFallbacks();
 };
 
+tests['getenv.stopError() should disable errors'] = function() {
```
Comment (ctavan) [inline test/fallbacks.js]: The test description doesn't match what's tested.

**Comment by ctavan** [comment]
@enVolt thanks for fixing! I discovered two more minor issues. Could you fix them? I'll merge afterwards.

**Comment by enVolt** [comment]
Thanks for your input. I've updated the same commit. Please check.

**Comment by nguyeda** [comment]
Hi guys, this is exactly the answer to a question I was about to ask. @ctavan do you already know if this is going to be merged? if so, is there an ETA on the 0.7 release?

Thanks for the great job 👍

**Comment by ctavan** [comment]
Has been published as getenv@0.7.0

Thanks to @enVolt for the contribution.

**Comment by nguyeda** [comment]
@ctavan that's awesome thanks a lot 👍

**Pull request merged**